etcd-configuration.adoc

```diff
@@ -0,0 +1,51 @@
+// vim: tw=79
+= etcd installation and configuration for Tendrl
+
+== etcd
+etcd is an open-source distributed, consistent key value store that provides a
+reliable way to store and access the data for shared configuration and service
+discovery across a cluster. Tendrl is using etcd to store the yaml files, other
+configuration files and its data.
```

Comment:
We are not storing the yaml files and config files right ? 
Can we have some thing like,"tendrl uses etcd as its central store to persist all the sds information, configuration definitions and tendrl management data".